### PR TITLE
LPS-126479

### PR DESCRIPTION
**Pull request by mlan0**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -3658,6 +3658,55 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-126479."
+	@priority = "4"
+	test SubmitFormWhenSessionExpires {
+		property portal.acceptance = "false";
+		property portal.upstream = "true";
+		property test.name.skip.portal.instance = "Forms#SubmitFormWhenSessionExpires";
+		property web.xml.timeout = "2";
```
Comment (mlan0) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:3667]: @natocesarrego , we still need to run this test to check if the timeout property is working.

**Comment by natocesarrego** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase (hunk 1 above)
Comment (natocesarrego) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:3667]: Indeed, this is not working. I've also run other Poshi test with the same property and it's failed. :disappointed: 

Therefore, I'll remove the Poshi test from this pull and forward it.

Thanks

**Comment by natocesarrego** [comment]
Resent without Poshi test: https://github.com/natocesarrego/liferay-portal/pull/1096

**Pull request closed without merging**